tests/test-data/fragment-html/sugar-img-test-fragment.html.out

```diff
@@ -22,12 +30,21 @@ Transformations made from HTML tags to AMP custom tags
 <img src="https://upload.wikimedia.org/wikipedia/commons/b/b7/True_Color_GIF_image-Tc217.gif"> at line 5
  ACTION TAKEN: img tag was converted to the amp-anim tag.
 
+<blockquote class="imgur-embed-pub" lang="en" data-id="UY7mHHT"> at line 8
+ ACTION TAKEN: blockquote.imgur imgur tag was converted to the amp-img tag
+
 
 AMP-HTML Validation Issues and Fixes
 -------------------------------------
-PASS
+FAIL
```

Comment:
I had to replace `PASS` with `FAIL` after adding the imgur test, which includes a `<script>` tag that the library removes. `FAIL` doesn't mean the unit test failed, in this case it means there was an AMP-Validation problem that was resolved by removing the tag (instead of converting it).